pkg/liqo-controller-manager/virtualNode-controller/suite_test.go

```diff
@@ -125,24 +134,61 @@ var _ = BeforeSuite(func(done Done) {
 
 	nms = &mapsv1alpha1.NamespaceMapList{}
 
-	technicalNamespace = &corev1.Namespace{
+	technicalNamespace1 = &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: technicalNamespaceNameID1,
+		},
+	}
+
+	technicalNamespace2 = &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: technicalNamespaceNameID2,
+		},
+	}
+
+	foreignCluster1 = &discoveryV1alpha1.ForeignCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: remoteClusterId1,
```

Comment:
```suggestion
			Name: remoteClusterID1,
```

If and when we will turn on the linter on tests, this will cause issues